libraries/chain/include/eosio/chain/unapplied_transaction_queue.hpp

```diff
@@ -96,15 +96,15 @@ class unapplied_transaction_queue {
       return itr->trx_meta;
    }
 
-   template <typename Func>
-   bool clear_expired( const time_point& pending_block_time, const time_point& deadline, Func&& callback ) {
+   template <typename Yeild, typename Callback>
```

Comment:
Probably should be `Yield` (both type and parameter name)

Reply:
If only I could spell. Thanks!